appserver/tests/appserv-tests/devtests/connector/v3/multiple-resources.xml/app/src/client/WebTest.java

```diff
@@ -42,13 +43,12 @@ public static void main(String args[]) {
         stat.printSummary(TEST_NAME);
     }
 
-private static void goGet(String host, int port,
-                              String result, String contextPath) {
+    private static void goGet(String host, int port, String result, String contextPath) {
 
-        try{
+        try {
             long time = System.currentTimeMillis();
             Socket s = new Socket(host, port);
-            s.setSoTimeout(2000000);
+            s.setSoTimeout(4000000);
```

Comment:
4000 seconds, 67 minutes? If the test should fail, shouldn't it fail much faster?